pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -306,6 +306,12 @@ spec:
           timeAdded: {{ .TimeAdded }}
 {{- end }}
 {{- end }}
+{{- end }}
+{{- with .kubeadmSkipPhases }}
+      skipPhases:
+      {{- range . }}
+      - {{ . }}
+      {{- end }}
```

Comment:
Curious. Why was this needed?

Reply:
Why is `skipPhases` needed or why is `end`?

Reply:
`skipPhases`